app/commands/action_command.rb

```diff
@@ -0,0 +1,50 @@
+# Add desc
+class ActionCommand
+  attr_reader :env
+  attr_reader :slashdeploy
+
+  def self.call(env)
+    new(env, ::SlashDeploy.service).call
+  end
+
+  def initialize(env, slashdeploy)
+    @env = env
+    @slashdeploy = slashdeploy
+  end
+
+  def call
+    logger.with_module(self.class) do
+      logger.info('running command')
+      run
+    end
+  end
+
+  def run
+    callback_id = action_payload.request.callback_id
+    message_action = MessageAction.find_by_callback_id(callback_id)
+    Object.const_get(message_action.command).call(env.merge('params' => message_action.command_params.to_h))
```

Comment:
Doing something like this always worries me from a security perspective; if somehow someone managed to inject a string into `message_action.command`, then they could `.call` anything they wanted.

Probably better to have a registry that actions can register themselves within, and then we get the action from that. Like:

```ruby
Actions.register 'Deploy', DeployAction
```

```
message_action.action_name
=> 'Deploy'
message_action.action
=> DeployAction
```